workflow-dev/tpl/generate_params.toml

```diff
@@ -45,6 +45,17 @@ logger_redis_location = "on-cluster"
 # - gcr: Use Google's GCR
 registry_location = "on-cluster"
 
+# Set the default (global) way of how Application (your own) images are
+# pulled from within the Controller.
+# This can be configured per Application as well in the Controller.
+#
+# This affects pull apps and git push (slugrunner images) apps
+#
+# Values values are:
+# - Always
+# - IfNotPresent
+image_pull_policy = "Always"
```

Comment:
it would be better  to have it as part of the controller as it is related to only it